dnf-behave-tests/fixtures/specs/system-upgrade-f30/pkg-b-2.0-1.spec

```diff
@@ -0,0 +1,15 @@
+Name:           pkg-b
+Version:        1.0
```

Comment:
The file name should be ``pkg-b-1.0-1.spec``.

Reply:
Fixed, thanks.